### PR TITLE
Add util function get_strict_version

### LINKED ISSUE #1685: Version info
The develop version has two entries of `__version__` in `aiida.__init__`.

Also, can we consider to implement some kind of version_info or similar that returns some standardized version number, for instance like a tuple. For instance using the distutils LooseVersion or StrictVersion.

### PR DESCRIPTION
**Pull request by sphuber**
Fixes #1685 

The utility function `aiida.get_strict_version` will return a StrictVersion
instance from `distutils.version` with the current version of the package.

### CODE REVIEW

**Comment by sphuber** [comment]
Note that this also fixes the duplication of the `__version__` definition in `aiida.__init__.py` (thanks @espenfl ).

**Comment by sphuber** [comment]
Let's wait for approval of either @DropD and @giovannipizzi as well to be sure

File: aiida/__init__.py [hunk 1]
```diff
@@ -8,13 +8,14 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 import warnings
+from distutils.version import StrictVersion
```
Comment (giovannipizzi) [inline aiida/__init__.py]: Do we want to have StrictVersion available at the `aiida` level?
Personally I prefer to hide it, so put it into the function.
But if you instead it's ok, I'm ok with a merge.

Comment (sphuber) [reply aiida/__init__.py]: I think that is a good point, I will move it in

**Pull request merged**